drivers/console/CMakeLists.txt

```diff
@@ -7,4 +7,4 @@ zephyr_sources_if_kconfig(uart_mcumgr.c)
 zephyr_sources_if_kconfig(uart_pipe.c)
 zephyr_sources_if_kconfig(telnet_console.c)
 zephyr_sources_if_kconfig(xtensa_sim_console.c)
-zephyr_sources_if_kconfig(native_posix_console.c)
```

Comment:
why are you removing the native_posix console?

Reply:
Some error while rebasing, @rveerama1 will send a fixed version shortly

Reply:
I have been maintaining websocket patches in RPL BR sample. It was my mistake in rebasing things. I will fix it. Thanks @nashif

Reply:
Fixed and pushed updated version.